test/components/user_list_item_test.js

```diff
@@ -30,4 +31,24 @@ describe("UserListItem", () => {
       expect(wrapper.text()).to.match(/treezy \(facilitator\)/i)
     })
   })
+
+  describe("user pictures", () => {
+    const user = {
+      given_name: "treezy",
+      online_at: 803,
+      is_facilitator: true,
+      picture: "http://some/image.jpg"
+    }
+
+    it("renders a list item that does have a picture", () => {
+      const wrapper = shallow(<UserListItem user={user} />)
+      expect(wrapper.find('img.picture').html())
```

Comment:
correction: these would only pass *if* enzyme is able to `find` an image tag, but i'd rather be explicit in the expectation/matcher combo than just relying on expect